### ISSUE TITLE
Repeated block of RLV calls during initial reset

### ISSUE THREAD

**Issue by emmamei** [emmamei/cdkey#588]
During the initial reset, the following block is repeated twice:
```
[09:13:56] Dolly Lady's Key:  executes: @sendchannel:75=rem (unset)
[09:13:56] Dolly Lady's Key:  executes: @accepttp=y (unset),tplm=y (unset),tploc=y (unset),fly=y (unset),unsit=y (unset),sit=y (unset)
[09:13:56] Dolly Lady's Key:  executes: @unsharedwear=y (unset),unsharedunwear=y (unset),attachallthis:=y (unset),detachallthis:=y (unset)
```
Then after a clearing block, that block is done another 5 times, then once the channel number is set (after reading preferences) the block is repeated once more (with appropriate channel).

This is part of issue #548.

**Comment by emmamei** [comment]
What we are seeing here is the evidence of a repeated use of the function `activateRLVBase()` in CheckRLV.lsl which has three calls to RLV, and sends out the config variable `defaultBaseRLVcmd`.

Shouldn't need that many calls to `activateRLVBase()`.

**Comment by emmamei** [comment]
This problem is limited to CheckRLV.lsl, and is the result of many different configuration variables being set. When these variables are set, `activateRLVBase()` is called:
- `autoTP`
- `afk`
- `canSelfTP`
- `canFly`
- `canStand`
- `canSit`
- `collapsed`
- `wearLock`

Several of these would not affect the settings in RLV at all.

**Comment by emmamei** [comment]
This has been fixed. The appropriate code has been added in order to selectively set and clear items rather than a whole block approach. Clothing lockdown was segregated into its own function; this has problems but will be resolved as part of issue #590.
